models/event.ts

```diff
@@ -7,10 +7,33 @@ export default class Event {
     name: string
     type: EventType
     description: string
-    startDate: Date
-    endDate: Date
+    address: string
+    startDate: Timestamp
+    endDate: Timestamp
     photo: Photo
-    postTime: Timestamp 
+    postTime: Timestamp
     stars: number
     votes: number
+    location: any
+    latitude: number
+    longitude: number
+
+    constructor(id: string, name: string, type: EventType, description: string, startDate: Timestamp, endDate: Timestamp, photo: Photo, postTime: Timestamp, stars: number, votes: number, location: any, latitude: number, longitude: number, address: string) {
```

Comment:
W swoim pr przerzuciłem się z używania class na interface. Wówczas nie musimy robić contructorów i ogółem jest to lepsze do obiektów jedynie dla danych